airflow/jobs/backfill_job.py

```diff
@@ -701,7 +705,7 @@ def tabulate_ti_keys_set(ti_keys: Iterable[TaskInstanceKey]) -> str:
             err += "Some task instances failed:\n"
             err += tabulate_ti_keys_set(ti_status.failed)
         if ti_status.deadlocked:
-            err += "BackfillJob is deadlocked."
+            err += "BackfillJobRunner is deadlocked."
```

Comment:
I wonder if we should just say the _backfill_ is deadlocked.

Reply:
Yep. Solved that one during rebasing on top of your typing change.